docs/admin-api/management-api.md

```diff
@@ -32,6 +27,12 @@
   - x-minio-operation: status
   - Response: On success 200, return json formatted StorageInfo object.
 
+* SetCredentials
+  - GET /?service
+  - x-minio-operation: setCredentials
```

Comment:
> You missed this @vadmeste

yes thanks. I am also going to add errors description that could be returned by this API.